doc/manipulating_visualizing/plotting.rst

```diff
@@ -40,6 +40,9 @@ different heuristics to find cutting coordinates.
      :target: ../auto_examples/manipulating_visualizing/plot_demo_plotting.html
      :scale: 50
 
+.. |plot_prob_atlas| image:: ../auto_examples/manipulating_visualizing/images/plot_prob_atlas_006.png
```

Comment:
Your example generates 3 images. So `plot_prob_atlas_006.png` will not exist. Pick an image from `plot_prob_atlas_001.png`, `plot_prob_atlas_002.png` or `plot_prob_atlas_003.png`.